site/content/docs/5.2/components/dropdowns.md

```diff
@@ -801,6 +842,11 @@ Use `data-bs-offset` or `data-bs-reference` to change the location of the dropdo
 
 By default, the dropdown menu is closed when clicking inside or outside the dropdown menu. You can use the `autoClose` option to change this behavior of the dropdown.
 
+<!-- Boosted mod : design callout -->
```

Comment:
A suggestion of how to do differently ?

Reply:
New version:
These 3 last **auto close behavior** variants should not be used because featuring a single selection, they do no respect the Orange Design System specifications. They can be used only if featuring multiple selections.